doc/user_guide/transformations.rst

```diff
@@ -271,6 +271,12 @@ can be found in the API-specific sections).
 
 ####
 
+.. autoclass:: psyclone.psyir.transformations.LoopTiling2DTrans
```

Comment:
Please could you move this so that it is in alphabetical order with the others (although I note that the loop chunking transformation is also in the wrong place).

Reply:
Moved both